tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php

```diff
@@ -29,27 +29,41 @@
 use Behat\Gherkin\Node\TableNode;
 use Configuration;
 use PHPUnit\Framework\Assert as Assert;
+use PHPUnit_Framework_Assert;
```

Comment:
```suggestion
use PHPUnit\Framework\Assert;
```

Reply:
I see, there is Assert :slightly_smiling_face: